Documentation/api.md

```diff
@@ -81,6 +81,7 @@ Endpoint defines a scrapeable endpoint serving Prometheus metrics.
 | interval | Interval at which metrics should be scraped | string | false |
 | tlsConfig | TLS configuration to use when scraping the endpoint | *[TLSConfig](#tlsconfig) | false |
 | bearerTokenFile | File to read bearer token for scraping targets. | string | false |
+| honorLabels | HonorLabels does not override labels from discovery on label name collisions when scraping. | bool | false |
```

Comment:
Mh, the wording seems a bit off. Sounds like it preservers target labels. But it's the opposite, if set to true, it does not override metric labels with target labels.